src/example/algdatastruct/lesson2/Main.java

```diff
@@ -0,0 +1,81 @@
+package example.algdatastruct.lesson2;
+
+import java.util.Arrays;
+import java.util.Random;
+
+public class Main {
+	private static int SIZE_OF_ARRAY = 100;
+	
+	public static void main(String[] args) {
+		
+		System.out.println(SIZE_OF_ARRAY);
+		/*
+			Задание 2.1
+		На основе программного кода из домашнего задания №1 реализуйте
+		массив на основе существующих примитивных или ссылочных типов
+		данных.
+		Выполните обращение к массиву и базовые операции класса Arrays.
+		Оценить выполненные методы с помощью базового класса
+		System.nanoTime().
+		*/
+		Task2_1(fillArrayRandomly(new int[SIZE_OF_ARRAY]));
+		
+		/*
+			Задание 2.2
+		На основе программного кода из домашнего задания №1 реализуйте
+		массив на основе существующих примитивных или ссылочных типов
+		данных.
+		Выполните обращение к массиву и базовые операции класса Arrays.
+		Оценить выполненные методы с помощью базового класса
+		System.nanoTime().
+		*/
+		
+	}
+	
+	
+	private static void Task2_1(int[] testArray) {
+		long time1;
+		long time2;
+		int searchKey = 13;
+		
+		
+		System.out.printf("\t\tTask 2.1.\t");
+		System.out.printf("Basic operations with class 'Arrays'.\n");
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [FS: Format string should use %n rather than n (VA_FORMAT_STRING_USES_NEWLINE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#fs-format-string-should-use-n-rather-than-n-va-format-string-uses-newline).
This format string includes a newline character (\n). In format strings, it is generally  preferable to use %n, which will produce the platform-specific line separator.